arches/management/commands/bulk_approve.py

```
@@ -0,0 +1,54 @@
+        )
+
+    def handle(self, *args, **options):
+        user_ids = options.get("user_ids")
+
+        if not user_ids:
+            raise CommandError("You must provide at least one user_id argument.")
+
+        User = get_user_model()
+        for user_id in user_ids:
+            try:
+                user = User.objects.get(pk=user_id)
```

Ruff (0.12.2)

Local variable `user` is assigned to but never used

Remove assignment to unused variable `user`

(F841)